latex/expressingParallelism.tex

```diff
@@ -2249,6 +2249,66 @@ \subsubsection{Defining kernels using OpenCL C kernel objects}
 \codeinline{cl_kernel} is no longer needed in the calling context.
 
 
+%*******************************************************************************
+% OpenCL Kernel Conventions and SYCL
+%*******************************************************************************
+\subsection{OpenCL Kernel Conventions and SYCL}
+\label{sec:opencl_kern_conventions_and_sycl}
+
+OpenCL and SYCL use opposite conventions for the unit stride dimension.  SYCL
+aligns with C++ conventions, which is important to understand from a performance
+perspective when porting code to SYCL.
+
+When specifying a \codeinline{range} as the global or local size
```

Comment:
This wording suggests that this only applies to the interop `parallel_for`, so the non-interop `parallel_for` API is free to choose how to map the unit stride, but the table further down appears to specify a mapping for non-interop SYCL kernels.

Reply:
I added text before the table describing when the columns apply.  The mapping reversal only applies with OpenCL interop kernels, while the SYCL column is for reference (already well defined in the spec).

Reply:
This is much clearer now, thanks.